space-framework/pkg/space-manager/provider.go

```diff
@@ -182,21 +184,28 @@ func (p *provider) processProviderWatchEvents() {
 				space.Name = spaceName
 				space.Spec.SpaceProviderDescName = p.name
 				space.Spec.Type = spacev1alpha1apis.SpaceTypeUnmanaged
-				space.Status.SpaceConfig = event.SpaceInfo.Config
+				err := p.createSpaceSecrets(&space, event.SpaceInfo)
+				if err != nil {
+					logger.Error(err, "Failed to create secrests for space "+p.name)
```

Comment:
Space can't be ready if we failed to create secrets.

Reply:
we check error on the top and don't do update